extension/user/getUser.js

```diff
@@ -0,0 +1,23 @@
+const {EACCESS, ENOTFOUND, customError} = require('../error')
+/**
+ * @param {SDKContext} context
+ * @param {Object} input
+ * @param {function} cb
+ */
+module.exports = (context, input, cb) => {
+  if (!context.meta.userId) {
+    return cb(customError(EACCESS, 'Unauthorized access'))
+  }
+
+  context.storage.extension.get(context.meta.userId, (err, user) => {
+    if (err) {
+      return cb(err)
+    }
+
+    if (!user) {
+      return cb(customError(ENOTFOUND, 'User not found'))
```

Comment:
Since this is not a search, the response text should be more definitive, like : "The given user does not exist."